.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "mdai-gateway (GoLand)",
+  "build": { "dockerfile": "Dockerfile", "context": ".." },
+  "workspaceFolder": "/workspaces/mdai-gateway",
+
+  "mounts": [
+    "type=volume,source=go-mod-cache,target=/home/goland/go/pkg/mod",
+    "type=volume,source=go-build-cache,target=/home/goland/.cache/go-build"
+  ],
+
+  "postCreateCommand": "go mod download && go get -tool github.com/golangci/golangci-lint/v2/cmd/golangci-lint@v2.4.0 && chown -R goland:goland /home/goland",
```

Comment:
Might be better to have these in the Dockerfile, so that we don't have to dig into the post create command to find the golangci-lint version